app/assets/stylesheets/application.scss

```diff
@@ -42,3 +42,21 @@
 .home-form {
   color: #ffffff;
 }
+
+.entries {
+  list-style: none;
```

Comment:
Properties should be ordered background-color, border, list-style, margin, text-align